.github/workflows/scripts/pentest_report.py

```
@@ -34,7 +34,13 @@ def norm_sev(s):
 def load_zap(path):
     if not path or not os.path.exists(path):
         return []
-    doc = json.load(open(path))
+    try:
+        doc = json.load(open(path))
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)